### PR TITLE
VrindaIssue768

### LINKED ISSUE #768: Editing garden items requires page refresh for changes to show up
Minor bug: When editing items in my garden, after clicking save changes, the data fields are blank until I refresh the page.

A tertiary non-bug is that the success popups appear to be per column instead of one event, which is kind of confusing but doesn't break anything. Screenshots attached.
screenshot from 2016-07-15 15-45-43
screenshot from 2016-07-15 15-46-13

### PR DESCRIPTION
**Pull request by TheChirpyWitch**
Closed #768

### CODE REVIEW

File: app/assets/javascripts/users/gardens.js [hunk 1]
```diff
@@ -22,6 +22,8 @@ openFarmApp.controller('gardenCtrl', ['$scope', '$http', '$rootScope',
           function(success, response) {
             if(success) {
               $scope.profileUser.gardens = response;
+              console.log(response);
```
Comment (simonv3) [inline app/assets/javascripts/users/gardens.js]: Hey @TheChirpyWitch Can you clean up these log statements? Or do you need them for debugging on the server?

**Comment by simonv3** [comment]
Looks good, awesome work! Does this close the issue? 

I've left a comment. 

In the future, even if you mention the issue, can you add a short statement about what you actually did to fix the issue? It makes tracking the commits easier in the commit history :).

File: app/assets/javascripts/users/gardens.js (hunk 1 above)
Comment (TheChirpyWitch) [inline app/assets/javascripts/users/gardens.js]: Hey @simonv3 ! I have added a new commit with the console.log statements removed and a description added. Coveralls seems okay with it. :dancer:

**Comment by simonv3** [comment]
:thumbsup:

**Pull request merged**